themes/bootstrap3/templates/search/results.phtml

```diff
@@ -115,7 +115,7 @@
 <?php /* End Main Listing */ ?>
 
 <?php /* Refine Search Options */ ?>
-<div class="<?=$this->layoutClass('sidebar')?>" id="search-sidebar">
+<div class="<?=$this->layoutClass('sidebar')?>" id="search-sidebar" role="region" aria-label="<?=$this->transEsc($this->slot('side-facet-caption')->get('Refine Results')) ?>">
```

Comment:
The side recommendation area is entirely configurable. It _usually_ includes the "refine results" facet controls, but in theory it could be configured to contain entirely different types of controls. In the scenario where the region contains "refine results" controls, it seems this label will be redundant. In the case where it contains something completely different, it seems that this label will be misleading. Do we really need a label for this at all? If each of the controls within the sidebar has an appropriate heading on it, won't that be sufficiently useful navigation without wrapping it in an extra layer? If the extra layer really is needed, maybe it needs a much broader label.

Reply:
`role="region"` demands an label according to the documents i have read. I already suspected that the content is to volatile for this kind of change, in that case, this cannot be done and its probably best removed.

Reply:
Yes, probably easiest to remove this for now. Maybe it's worth reviewing all the templates in the "Recommend" directory to make sure they all have appropriate headings. I think as long as all of those are well-labeled, the sidebar contents should be perfectly accessible.